src/js/helpers/covid19Helper.js

```diff
@@ -177,13 +177,19 @@ export const handleSort = (a, b) => {
     return 0;
 };
 
+const calculateDifferenceSafely = (x, y) => {
```

Comment:
I like the idea of this function. In the future, we could use this elsewhere, and we should handle zero cases.